src/main/java/mate/jdbc/service/impl/ManufacturerServiceImpl.java

```diff
@@ -0,0 +1,46 @@
+package mate.jdbc.service.impl;
+
+import java.util.List;
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.ManufacturerService;
+
+@Service
+public class ManufacturerServiceImpl implements ManufacturerService {
+    private final Injector injector = Injector.getInstance("mate.jdbc");
+    private final ManufacturerDao manufacturerDao;
+
+    {
+        manufacturerDao = (ManufacturerDao) injector.getInstance(ManufacturerDao.class);
+    }
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        return manufacturerDao.create(manufacturer);
+    }
+
+    @Override
+    public Manufacturer get(Long id) {
+        if (manufacturerDao.get(id).isPresent()) {
+            return manufacturerDao.get(id).get();
+        }
+        throw new RuntimeException("Can't find driver with such id: " + id);
```

Comment:
let's refactor this in functional style. use orElseThrow() method when getting a value from the Optional.